CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ IMPROVEMENTS:
 * CRDs: add value `controller.aclToken` to support manually passing in an ACL token to the CRD controller if independently managing ACLs. [[GH-783](https://github.com/hashicorp/consul-helm/pull/783)]
 * TLS: Consul client certificates now include their pod IPs in the IP SANs. This applies to auto-encrypt enabled and disabled. [[GH-805](https://github.com/hashicorp/consul-helm/pull/805)]
 * Consul client nodes have a new meta key called "host-ip" set to the IP of the Kubernetes node they're running on. [[GH-805](https://github.com/hashicorp/consul-helm/pull/805)]
+* Connect: the latest version of consul-k8s cleans up Consul service instances whose pods are no longer running.
```

Comment:
meta comment:  I know that we start the line with  `"Connect:"` but do you think this is sufficiently descriptive to a random reader that this is ONLY for connect injected pods, not sync-catalog?

Reply:
No probably not, will update to make it clear. Something like `cleans up Consul connect service instances...`?

Reply:
just unresolving so I don't forget.